js/fizzbuzz.js

```diff
@@ -0,0 +1,71 @@
+// scope FizzyBuzzy for privacy
+var FizzyBuzzy = (function() {  
+  // defining _FizzBuzz as a constructon function
+  var _FizzBuzz = function(word1,word2) {
+    this.word1 = word1 || 'Fizz';
+    this.word2 = word2 || 'Buzz';
+  }; 
+  
+  // PRIVATE function, not a method on _FizzBuzz
+  // start and stop come from the READ function where doFizzBuzz is called
+  function doFizzBuzz(start,stop,word1,word2) {  
+    // array to store the results
+    var resultArr = [];
+    // for loop to generate results
+    for (var i = start; i < stop + 1; i++) {
+      if (i % 3 === 0 && i % 5 !== 0) {
+        resultArr.push(word1); 
+      } else if (i % 5 === 0 && i % 3 !== 0) {
+        resultArr.push(word2); 
+      } else if (i % 3 === 0 && i % 5 === 0) {
+        resultArr.push(word1 + word2); 
+      } else {
+        resultArr.push(i); 
+      }
+    }
+    // return the array so it can be assigned to this.result in READ
+    return resultArr; 
+  };
+  
+  _FizzBuzz.prototype.read = function(start,stop) {
+    // called in the event listener, 
+    // given the values entered in the form as start and stop
+    this.start = start;
+    this.stop = stop + 1;
+    // pass values into doFizzBuzz
+    this.result = doFizzBuzz(start,stop,this.word1,this.word2);
```

Comment:
You're consistent with this, which I appreciate, but it might be a little more readable if you used spaces after your commas (especially since you're passing in properties of the `this` object):

`this.result = doFizzBuzz(start, stop, this.word1, this.word2);`